.github/workflows/playwright.yml

```
@@ -0,0 +1,41 @@
+name: Playwright Tests
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  test-playwright:
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '17.x'
+          cache: 'npm'
+      - name: Install dependencies
+        run: npm install
```

On CI, it is recommended to use `npm ci` since that doesn't update config files.

```suggestion
        run: npm ci
```